CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 ### Bugfixes
 
 - [#3180](https://github.com/influxdb/influxdb/pull/3180): Log GOMAXPROCS, version, and commit on startup.
+- [#3193](https://github.com/influxdb/influxdb/pull/3193): Fix two panics
```

Comment:
I suggest we enhance this note a little -- it's not much use to someone looking to see if the panic they are experiencing was fixed. Simply "Fix panic for SHOW STATS and in collectd" would be much better.